hydration_context/src/ssr.rs

```diff
@@ -58,6 +58,25 @@ impl SsrSharedContext {
             ..Default::default()
         }
     }
+
+    /// Consume the data buffers, awaiting all async resources,
+    /// returning both sync and async buffers.
+    /// Useful to implement custom hydration contexts.
+    /// This can only be called once.
```

Comment:
Probably worth clarifying this comment; it takes `&self`, so it certainly can be called more than once! Maybe explain what happens if you call it more than once, instead?

Reply:
😆 that's true. Updated!